opencti-platform/opencti-front/src/private/components/workspaces/dashboards/WidgetConfig.jsx

```diff
@@ -1080,116 +1078,128 @@ const WidgetConfig = ({ workspace, widget, onComplete, closeMenu }) => {
+                                        ),
+                                      )}
+                                    </MenuItem>
+                                  ))}
+                                </Select>
+                              );
+                            }
+                            return <div/>;
+                          }}
+                        />
+                      </FormControl>
+                      )}
                     {dataSelection[i].perspective === 'entities'
-                                                && getCurrentSelectedEntityTypes(i).length === 0 && (
-                                                <TextField
-                                                  style={{
-                                                    flex: 1,
-                                                    marginRight:
-                                                                dataSelection[i].perspective === 'relationships'
-                                                                  ? 20
-                                                                  : 0,
-                                                  }}
-                                                  label={t_i18n('Field')}
-                                                  fullWidth={true}
-                                                  value={dataSelection[i].attribute}
-                                                  placeholder={t_i18n('Series attribute')}
-                                                  onChange={(event) => handleChangeDataValidationParameter(
-                                                    i,
-                                                    'attribute',
-                                                    event.target.value,
-                                                  )
-                                                        }
-                                                />
+                      && getCurrentSelectedEntityTypes(i).length === 0 && (
+                        <Select
+                          fullWidth={true}
+                          value={dataSelection[i].attribute ?? 'entity_type'}
+                          onChange={(event) => handleChangeDataValidationParameter(
+                            i,
+                            'attribute',
+                            event.target.value,
+                          )
+                          }
+                        >
+                          {[
+                            { value: 'entity_type' },
+                            { value: 'created-by.internal_id' },
+                            { value: 'object-label.internal_id' },
+                            { value: 'object-assignee.internal_id' },
+                            { value: 'object-marking.internal_id' },
+                            { value: 'kill-chain-phase.internal_id' },
+                            { value: 'x_opencti_workflow_id' },
```

Comment:
I suppose it is the list of common attributes no matter the entity type?

A detail, you could have used a simple array of string instead of using objects 😁

Reply:
Yes it is :)

Reply:
Indeed, I just kept the same format as used for the other widget types

Reply:
going to fix it :)